OfficeMicroService/Presentation/Controllers/OfficeController.cs

```diff
@@ -0,0 +1,129 @@
+﻿using IdentityMicroService.Domain.Entities.Enums;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using OfficeMicroService.Application.Services;
+using OfficeMicroService.Data.Enum;
+using OfficeMicroService.Data.Models.DTO;
+using Serilog;
+
+namespace OfficeMicroService.Presentation.Controllers
+{
+    [Route("[controller]/[action]")]
+    [ApiController]
+    public class OfficeController : Controller
+    {
+        private readonly IOfficeServices _officeServices;
+
+        public OfficeController(IOfficeServices officeServices)
+        {
+            _officeServices = officeServices;
+        }
+
+        /// <summary>
+        /// Get All Offices
+        /// </summary>
+        /// <response code="200">Returns offices list</response>
+        /// <response code="500">Operation wasn't succeeded</response>
+        [HttpGet]
+        public async Task<IActionResult> GetAll()
```

Comment:
create a separate dto to fit the confluence

Reply:
i dont understand...